DataDefinitions/LegalStatus.cs

```diff
@@ -24,6 +24,8 @@ static LegalStatus()
             var Hunter = new LegalStatus("Hunter");
         }
 
+        public static readonly LegalStatus None;
```

Comment:
Maybe `Clean` should be the default legal status instead of `None`?

Reply:
Until the status is known, ‘None’ is a better descriptor, IMO

Reply:
Innocent until proven guilty? ;-)